packages/app-admin-ui/index.js

```diff
@@ -99,6 +103,8 @@ class AdminUIApp {
         adminMeta,
         entry: 'index',
         outputPath: path.join(builtAdminRoot, 'secure'),
+        enableDevFeatures: this._enableDevFeatures,
```

Comment:
What does this actually enable? I can't find any info in the webpack docs about it.

Reply:
I thought this was you 😆 It enables the 'update' button in the AdminUI kinda like a feature flag. I assume it's because the update button (which does batch updates) is not working for all field types or is an experimental feature or something?

I'd actually rather remove it or enable it across the board but if experimental features is something we want enabling via config is the right approach.

Reply:
If Update button does not work on every field type then we should disable those fields when in production mode rather than turning off this important feature. 

Moreover those non supported fields may be updated using custom action  example i have

Reply:
Ok so it looks like build should always be `production` and `createDevMiddleware` always `development`. Love it! Deleting code :)

Reply:
I checked with Joss (who added it) and it seem like indeed it is not 100% function and ready So I've renamed this flag `experimentalFeatures` and I've also removed the other reference to `ENABLE_DEV_FEATURES`.